tailcfg/tailcfg.go

```diff
@@ -525,6 +525,7 @@ type Hostinfo struct {
 	Desktop         opt.Bool       `json:",omitempty"` // if a desktop was detected on Linux
 	Package         string         `json:",omitempty"` // Tailscale package to disambiguate ("choco", "appstore", etc; "" for unknown)
 	DeviceModel     string         `json:",omitempty"` // mobile phone model ("Pixel 3a", "iPhone12,3")
+	DeviceToken     string         `json:",omitempty"` // macOS/iOS APNs device token for notifications (and maybe Android in the future)
```

Comment:
should we call this NotifyToken? MobileNotifyToken?

Reply:
it's called DeviceToken in APNs. I'm not super keen on inventing a new name.

Reply:
Well I was thinking that if we use it for Android too it'll be wrong for one platform at least so we should make up a common term for both.

But it seems like Android also uses "Device Token" so sure! Good as is.

Reply:
Still think that having something about "push" or "notification" in the name would be useful, since it's next to a bunch of other generic fields (`DeviceModel`, `Package`) and it's not clear what kind of token we're talking about.

Reply:
ok done!